README.md

```diff
@@ -482,20 +494,48 @@ import {
   StackNavigation,
 } from '@exponent/ex-navigation';
 
-import AppStore from './state/Store';
-
-const Router = createRouter(() => ({
-  home: () => HomeScreen,
-}));
+import Store from './state/Store';
+import Router from './Router';
 
 const navigationContext = new NavigationContext({
   router: Router,
   store: Store,
 })
 
 return (
-  <NavigationProvider context={navigationContext}>
-    <StackNavigation yourUsualPropsHere />
-  </NavigationProvider>
+  <Provider store={Store}>
+    <NavigationProvider context={navigationContext}>
+      <StackNavigation yourUsualPropsHere />
+    </NavigationProvider>
+  </Provider>
 )
 ```
+
+### Call route actions from redux-saga middleware
+
+Add your saga middleware like normally to redux store and then:
+
+```javascript
+/* In your saga middleware code */
+import { takeEvery } from 'redux-saga'
+import { put, select } from 'redux-saga/effects'
+import { NavigationActions } from '@exponent/ex-navigation'
+import Router from './Router'
+import selectors from './selectors'
+
+function* goHome() {
+  const navigatorUID = yield select(selectors.getNavigatorUID)
+  yield put(NavigationActions.push(navigatorUID, Router.getRoute('home')))
```

Comment:
OK, can you write an example how to use navigator outside components. There wasn't any instructions and this was the way @brentvatne aided me to do it in slack and he asked to do a pull request  for this.